src/store/keyboard/context.js

```diff
@@ -0,0 +1,19 @@
+import { createContext, useContext } from 'react';
+
+const keyboardContext = createContext();
+
+export const KeyboardContextProvider = ({ children, value }) => {
```

Comment:
I think `KeyboardProvider` would be better naming convention here to avoid confusion since we're already using `keyboardContext.Provider` below

Reply:
I think it should not be KeyboardProvider, because it is providing the context of that, and keyboardProvider would be more confusing in my opinion

Reply:
> I think it should not be KeyboardProvider, because it is providing the context of that, and keyboardProvider would be more confusing in my opinion

Please for check for reference: https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
thanks for sharing, making the changes